Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

Starts Wireshark via pfexec to leverage any given privileges.

--- wireshark.desktop
+++ wireshark.desktop
@@ -69,8 +69,8 @@
 Comment[fr]=Analyseur de trafic réseau
 Comment[sv]=Nätverkstrafikanalysator
 Icon=wireshark
-TryExec=wireshark
-Exec=wireshark %f
+TryExec=/usr/sbin/wireshark
+Exec=pfexec /usr/sbin/wireshark %f
 Terminal=false
 MimeType=application/vnd.tcpdump.pcap;application/x-pcapng;application/x-snoop;application/x-iptrace;application/x-lanalyzer;application/x-nettl;application/x-radcom;application/x-etherpeek;application/x-visualnetworks;application/x-netinstobserver;application/x-5view;
 # Category entry according to:
